en/user_guide/workflow.rst

```diff
@@ -0,0 +1,51 @@
+Workflow
+========
+
+Open orchestrated allows to have a publication workflow.
+It is a series of steps to achieve for publication of a node or a content before it can be displayed on Front Office.
+
+Status
+------
+
+Nodes and contents have status, change it allows to pass the series of steps of publication workflow.
+
+* **Initial status**: fist workflow status (status that a node or a content will have after its creation).
+* **Final status**: latest workflow status (status which allows to display on the Front Office).
+
+The users can define the status which will make up their workflow but it can have only one initial and final status.
+
+Status creation
+~~~~~~~~~~~~~~~
+
+* **Name**(required): status name
+* **Published**: if this status is final
+* **Initial**: if this status is initial
```

Comment:
maybe put a note after this list to quickly explain what is "initial" and "final"